mirage/config.js

```diff
@@ -1,8 +1,10 @@
 export default function() {
-  this.resource('books');
-  this.resource('pages');
-  this.resource('users');
-  this.passthrough('https://coolwriter.eu.auth0.com/**');
+  // //this.namespace = '/api/v1';
+
+  // this.resource('books');
```

Comment:
why removal?

Reply:
I did it because I have not realized that the config of `Mirage` is located in `environment.js`. But then you can see I changed it in the config file. Good catch @PoslinskiNet it should be uncommented and set up in the `test` section in the `environment.js`.